less_users/models.py

```diff
@@ -62,19 +47,34 @@ def get_points(self):
         and this challenge makes possible to get points once for every 7 days;
         for 1/day challenge points can be gained each day"""
         points = self.challenge.points
-        if len(self.log_set.all()) > 0:
-            last_log = int(self.log_set.last().date.strftime("%Y%m%d"))
-        else:
-            last_log = 0
         frequency = self.challenge.frequency
-        today = self.todays_day_num()
-        if today - last_log > frequency:
-            return points
-        return 0
+        self.check_if_active()
```

Comment:
dlaczego nie przekazujesz daty?

Reply:
wprowadziłam zmianę w funkcji check_if_active, i nie naniosłam poprawek w resztę kodu (wcześniej to było @property, bez dodatkowych argumentów).